en/modules/features/pages/components.adoc

```diff
@@ -1,94 +1,99 @@
 = Components
```

Comment:
I don't know if we actually want to maintain this "Features" section in the documentation. 
Just to give you context about this section @codePau - this was made by @carolromero and @xabier while we actually were starting to develop much of Decidim, and it was something more like a wishlist ("this is how we want that decidim works in the near future") rather than "this is how it works".   
Is this useful and briging value @carolromero?